### PR TITLE
Add basic parts to support LTE_CA RAT information

### PR DESCRIPTION
**Pull request by Flohack74**
See also https://git.sailfishos.org/mer-core/ofono/commit/c1156320acd67b8e1afeeb1c5ed581262f2edc66 - we are basically mapping LTE_CA to LTE for the moment. Later it would be perfectly okay to expose that information in the correct way to the consumers, and add a new ACCESS_TECHNOLOGY enum member for this.

TODO: Does anyone has verified access to LTE-CA so we can test this?

### CODE REVIEW

File: plugins/u8500.c [hunk 1]
```diff
@@ -127,6 +127,7 @@ static void set_power_by_mce_state(struct ofono_modem *modem,
 	case MCE_NORMAL:
 		if (isi->online_cbd)
 			report_online(isi, mce_state == MCE_NORMAL);
+		break;
```
Comment (peat-psuwit) [inline plugins/u8500.c]: I don't think we're supposed to put `break` here.
https://git.kernel.org/pub/scm/network/ofono/ofono.git/tree/plugins/u8500.c#n130

Although, we don't use this plugin anyway. What if we just disable building it?

**Review by peat-psuwit — CHANGES_REQUESTED** [review]

File: plugins/u8500.c (hunk 1 above)
Comment (Flohack74) [reply plugins/u8500.c]: Ok I will fix it. Well we can disable it, too, but I think fixing a compile issue even for disabled code is a god thing :)

**Comment by mariogrip** [comment]
So LTE_CA is LTE+, but do we have any devices with this version of ofono that support LTE+?  

https://github.com/aosp-mirror/platform_frameworks_base/blob/c5d02da0f6553a00da6b0d833b67d3bbe87341e0/packages/SystemUI/src/com/android/systemui/statusbar/policy/MobileSignalController.java#L271

**Comment by Flohack74** [comment]
Yeah thats a good question, I really dont know. Also, you need a matching carrier :) I will do a quick review what devices could have it.

**Comment by Flohack74** [comment]
grafik

grafik

grafik

Plenty of options ;)

**Review by fredldotme — APPROVED** [review]
LGTM

**Review by peat-psuwit — APPROVED** [review]

**Pull request merged**